### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by iamonuwa**

### CODE REVIEW

File: src/app/services/preloader.service.ts [hunk 1]
```diff
@@ -8,15 +8,14 @@ export class PreloaderService {
...
   public show() {
     this._element.style['display'] = 'block';
   }
 
-  public hide(delay: number = 0) {
-    setTimeout(() => {
+  public hide() {
       this._element.style['display'] = 'none';
-    }, delay);
   }
```
Comment (tkrevh) [inline src/app/services/preloader.service.ts]: Why was this change necessary @iamonuwa ?

File: src/app/services/preloader.service.ts [hunk 2]
```diff
@@ -8,15 +8,14 @@ export class PreloaderService {
 
   constructor() {
     this._element = document.getElementById(this._selector);
+    console.log(`Hello World`, this._element);
```
Comment (tkrevh) [inline src/app/services/preloader.service.ts]: Please remove

File: src/app/services/preloader.service.ts (hunk 1 above)
Comment (iamonuwa) [reply src/app/services/preloader.service.ts]: That `setTimeOut` set to 0 keeps the preloader active without adding the display none style to the dom
